README.md

```diff
@@ -1,72 +1,97 @@
-![](https://img.shields.io/badge/Microverse-blueviolet) 
+# HTML & CSS3 Course
 
-# Portfolio: setup and mobile version skeleton
+If you are not familiar with linters, read [root level README](../README.md).
 
-For this repository, I've created html templates and styling files in order to built a responsive portfolio ,  which can help to easy explorate  profiles from differents devices.
+## Set-up GitHub Actions
 
-## Built With 🔨
+Please do the following **steps in this order**:
 
-- HTML
-- CSS
-- Git & Github
+1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
+   - **Remember** to use the file linked above
```

Comment:
Please update your readme file to follow [microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md)